project/plugins.sbt

```diff
@@ -11,7 +11,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-bobby" % "3.5.0")
```

Comment:
If you are interested try to increase other library or plugin versions too like 
https://github.com/hmrc/sbt-bobby/releases/tag/v4.1.0

just give it a try and if not working you can leave it as is. I do the same thing when I do any library upgrade.

Reply:
I upgraded everything I could.